library/core/src/num/nonzero.rs

```diff
@@ -45,7 +47,9 @@ macro_rules! impl_zeroable_primitive {
             pub trait Sealed {}
 
             $(
-                #[derive(Debug, Clone, Copy, PartialEq)]
+                // This inner type is never shown directly, so intentionally does not have Debug
+                #[expect(missing_debug_implementations)]
+                #[derive(Clone, Copy)]
```

Comment:
So we keep clone in the derive because we know it desugared to using the copy impl instead of a field access? Seems worth w comment